src/higley_lab_to_nwb/benisty_2022/benisty_2022_convert_session.py

```diff
@@ -17,27 +17,17 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
         output_dir_path = output_dir_path / "nwb_stub"
     output_dir_path.mkdir(parents=True, exist_ok=True)
 
-    session_id = "subject_identifier_usually"
+    session_id = "11222019_grabAM05_spont"
     nwbfile_path = output_dir_path / f"{session_id}.nwb"
 
     source_data = dict()
     conversion_options = dict()
 
-    # Add Recording
-    source_data.update(dict(Recording=dict()))
-    conversion_options.update(dict(Recording=dict()))
-
-    # Add LFP
-    source_data.update(dict(LFP=dict()))
-    conversion_options.update(dict(LFP=dict()))
-
-    # Add Sorting
-    source_data.update(dict(Sorting=dict()))
-    conversion_options.update(dict(Sorting=dict()))
-
-    # Add Behavior
-    source_data.update(dict(Behavior=dict()))
-    conversion_options.update(dict(Behavior=dict()))
+    # Add Wheel signal
+    file_path = str(data_dir_path / session_id / f"{session_id}_spike2.smrx")
+    stream_id = '4'
```

Comment:
Is there a way to determine which `stream_id` corresponds to the wheel signal? 

For reference, I've seen an example where the neuro signals for TDT format are not always in the '0' stream, so I'd have to identify them based on name.

Reply:
Yes, actually I can! I already have two functions in the interface to get the stream id and the stream name, I will just combine the two to get the mapping.

Reply:
I would then just expose `stream_name` to the interface, and figure out which stream_id that corresponds to and init the extractor with that `stream_id`.